content/advanced/330_servicemesh_using_appmesh/integrate_observability/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Observability"
+date: 2020-01-27T08:30:11-07:00
+weight: 90
+draft: false
+---
+
+ In this chapter, we will dive into key operational metrics and tools such as CloudWatch Container Insights, Cloudwatch Logs and Prometheus that you can leverage within the AWS environment to supercharge your ability to monitor metrics, collect logs, trigger alerts, and trace distributed services.
+
+Observability includes
```

Comment:
"Observability includes" could maybe be more here, e.g. the use of various signals (metrics, traces, logs) to monitor the overall health of your application, etc

This, tie it together with something like "In this section, you'll use the following tools to..." etc

Reply:
Updated